tests/in/shadow.wgsl

```diff
@@ -1,61 +1,115 @@
 struct Globals {
+    view_proj: mat4x4<f32>;
     num_lights: vec4<u32>;
 };
 
-@group(0) @binding(0)
+@group(0)
+@binding(0)
 var<uniform> u_globals: Globals;
 
+struct Entity {
+    world: mat4x4<f32>;
+    color: vec4<f32>;
+};
+
+@group(1)
+@binding(0)
+var<uniform> u_entity: Entity;
+
+@stage(vertex)
+fn vs_bake(@location(0) position: vec4<i32>) -> @builtin(position) vec4<f32> {
```

Comment:
I don't think this entry point useful for testing. Perhaps, we could remove it?

Reply:
Make sense. I removed the entry point.

btw, I tried to remove `tests/out/glsl/*` and regenerate again. 
But `tests/out/glsl/quad.main.Fragment.glsl` and `tests/out/glsl/quad.main.Vertex.glsl` doesn't regenerated.
Maybe those are renamed to `vert_main` and `frag_main` but forgot to delete?

Reply:
yes that sounds right